packages/vm/src/instance.rs

```diff
@@ -520,7 +512,7 @@ mod tests {
 
         let report2 = instance.create_gas_report();
         assert_eq!(report2.used_externally, 146);
-        assert_eq!(report2.used_internally, 76371);
+        assert_eq!(report2.used_internally, 657571);
```

Comment:
10x gas price? Let's sort this out before tagging - definitely need to rework the conversion factor from sdk/wasmer gas again

Reply:
Jupp, this is a random value now. We need to define the gas cost properly.